SOURCES/cve-2019-1551.patch

```diff
@@ -0,0 +1,768 @@
+From 4ce070d65e6c4d55bcd6bdc12a4d99d9aac47fc7 Mon Sep 17 00:00:00 2001
+From: Lucas RAVAGNIER <lucas.ravagnier@vates.tech>
+Date: Tue, 29 Oct 2024 10:43:56 +0100
+Subject: [PATCH 3/3] Fix an overflow bug in rsaz_512_sqr
```

Comment:
Have we lost the original patch/commit author here? (this also applies to other patches)

Reply:
Fixed